src/components/TradesWidget/index.tsx

```diff
@@ -48,13 +49,28 @@ function csvTransformer(trade: Trade): CsvColumns {
     fillPrice,
     sellAmount,
     buyAmount,
+    orderSellAmount,
     timestamp,
     txHash,
     eventIndex,
     orderId,
     batchId,
   } = trade
 
+  const limitPriceStr = limitPrice ? formatPrice({ price: invertPrice(limitPrice), decimals: 8 }) : 'N/A'
+  const inverseLimitPriceStr = limitPrice ? formatPrice({ price: limitPrice, decimals: 8 }) : 'N/A'
+  const fillPriceStr = formatPrice({ price: invertPrice(fillPrice), decimals: 8 })
+  const inverseFillPriceStr = formatPrice({ price: fillPrice, decimals: 8 })
+
+  const fillPercentage =
+    orderSellAmount && trade.type != 'liquidity'
+      ? new BigNumber(sellAmount.toString()).dividedBy(new BigNumber(orderSellAmount.toString())).toString(10)
```

Comment:
I'm not sure,but do you not need to `* 100`?
If filled half of what the order is for: `sellAmount = 50, orderSellAmount = 100, sellAmount/orderSellAmount = 0.5`
I may be looking for the wrong numbers

Reply:
If we are thinking in terms of displaying the percentage nicely, yes.
But since this is going into a csv which will be loaded into a spreadsheet program, you can just chose to display the column as `%`, which will do the formatting for you. In that case, it would be worse if we multiply it by 100.

Example:
![screenshot_2020-07-10_08-59-49](https://user-images.githubusercontent.com/43217/87174173-a56f5100-c28b-11ea-957e-ea4c3f907754.png)
The % I'm talking about:
![screenshot_2020-07-10_08-59-58](https://user-images.githubusercontent.com/43217/87174206-b15b1300-c28b-11ea-8e49-a133401c9800.png)

Reply:
Is it a common practice to provide 0.5 and for users to switch to %? If I were to see a column `Fill %` with `0.5` value I would think it means `0.5%`

But if you say so, sure

Reply:
Well, that's my expectation of many years of expertise working tirelessly with spreadsheets.
Which is not a real thing. 

Anyway, that's what I think makes more sense to be represented math wise.